dask/dataframe/core.py

```
@@ -529,6 +529,22 @@ def clip(self, lower=None, upper=None):
     def notnull(self):
         return map_partitions(pd.Series.notnull, self.name, self)
 
+    def apply(self, func, convert_dtype=True, args=(), **kwds):
+        """ Parallel version of pandas.Series.apply
+
+        This is identical to the Pandas algorithm but assumes that axis=0.
```

Or we could raise an error asking them to specify `axis=0` for now.